src/app/shared/app-nav.component.html

```diff
@@ -1,11 +1,16 @@
-<sky-navbar>
-  <sky-navbar-item *ngFor="let n of nav">
-    <a
-      [routerLink]="n.path"
-      [routerLinkActiveOptions]="{exact: true}"
-      routerLinkActive="sky-navbar-item-active"
+<nav>
+  <ul class="app-navbar">
```

Comment:
Navbar module was ported; should we keep using it?
https://github.com/blackbaud/skyux-navbar